lib/internal/policy/manifest.js

```diff
@@ -354,6 +345,22 @@ function findScopeHREF(href, scopeStore, allowSame) {
  * @typedef {boolean | string | SRI[] | typeof kCascade} Integrity
  */
 
+let shouldAbortOnUncaughtException;
```

Comment:
It seems only the initialization of shouldAbortOnUncaughtException has been changed in this patch - is there a reason that these are moved down here? It might be better to keep them where they were for better git-blame results